features/step_definitions/salt_steps.rb

```diff
@@ -864,9 +864,10 @@
   end
 end
 
-Then(/^I run spacecmd listevents for sle-minion$/) do
+Then(/^I run spacecmd listevents for ([^\s]*)$/) do |minion|
+  fullhostname = get_target_fullhostname(minion)
```

Comment:
I renamed the `minion` variable to `system` as it doesn't need to represent a minion.